### PR TITLE
User adding item in cart

### PR DESCRIPTION
**Pull request by pacifiquemboni**
## 1.What does this PR do?
Users should be able to add product in the cart
# ⚠️⚠️How to test this PR
1. Clone `develop` branch and checkout to `user-adding-item-in-cart`
2. Install all required dependencies `npm install`
3. Follow `.env.example` to create your `.env` file
4. Start the application using `npm start`
5.  Navigate to login and start by signing up, verify your account 
Screenshot 2024-07-06 222043

### CODE REVIEW

**Comment by UmuhireJessie** [comment]
Can you make products available for testing purposes?
Screenshot 2024-07-08 at 08 57 51

**Review by lilika67 — APPROVED** [review]

**Review by UmuhireJessie — APPROVED** [review]

File: src/components/cart/cart.tsx [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+      dispatch(fetchProductsInCart());
+      dispatch(fetchTotalInCart());
+    } catch (error) {
+      console.error("Failed to clear the cart:", error);
+    }
+  };
+  return (
+    <div className="cart-body">
+      {loading ? (
+        <div>Loading...</div>
```
Comment (UmuhireJessie) [inline src/components/cart/cart.tsx]: Can you improve the design of the loading sign?

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
Check comments and please resolve the conflicts

**Comment by Celse13** [comment]
I have one more question, which is all about verifying if the preferred currency in the profile settings correlates with the currency being displayed on the cart

**Comment by Celse13** [comment]
I have one more question which is about verifying if the currency in the profile settings correlates with the currency being displayed on the cart.

File: .env.example [hunk 2]
```diff
@@ -1,7 +1,10 @@
 BACKEND_URL = 'your backend url'
 GOOGLE_CLIENT_ID='your google client id'
 REACT_APP_BACKEND_URL="YOUR_BACKEND_URL"
+<<<<<<< HEAD
 
+=======
+>>>>>>> a9b4a07 ([Delivers #187355068] user adding item in cart)
```
Comment (UmuhireJessie) [inline .env.example]: Remove these lines

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
Check my comments

**Review by igorihimbazwe — APPROVED** [review]

**Review by shema-surge — APPROVED** [review]

**Review by UmuhireJessie — APPROVED** [review]

**Pull request merged**